src/view/configuration/configuration.jsx

```diff
@@ -1030,40 +1023,35 @@ const Configuration = ({
                           </div>
                         </div>
                       </div>
-                      <div className="u-gapTop2x">
-                        <ModalTrigger>
-                          <Button
-                            data-test-id="deleteInstanceButton"
-                            label="Delete Instance"
-                            icon={<Delete />}
-                            variant="action"
-                            disabled={values.instances.length === 1}
-                          />
-                          {values.instances.length === 1 ? (
-                            <span className="Note u-gapLeft">
-                              You must have at least one instance to use this
-                              extension.
-                            </span>
-                          ) : null}
```

Comment:
Previously we were showing the Delete Instance button but if there was only one instance we would disable it and add this note next to it. I don't think it really improved the user experience to show and disable it and it cluttered the UI for the majority of cases where only one instance is configured. This was especially the case for IdentityMap, where we have identities and then identifiers nested inside each identity. It felt like button overload.